### PR TITLE
Us35 s04 admin dashboard

### PR DESCRIPTION
**Pull request by victorzanotelli**

### CODE REVIEW

**Review by Guadanino — APPROVED** [review]

**Review by R-Ztr — APPROVED** [review]

File: src/Controller/Admin/CategoryCrudController.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class CategoryCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return Category::class;
+    }
+
+    /*
+    public function configureFields(string $pageName): iterable
```
Comment (StoicFrenchDev) [inline src/Controller/Admin/CategoryCrudController.php:19]: dead code

File: src/Controller/Admin/UserCrudController.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class UserCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return User::class;
+    }
+
+    /*
+    public function configureFields(string $pageName): iterable
+    {
```
Comment (StoicFrenchDev) [inline src/Controller/Admin/UserCrudController.php:20]: dead code

File: src/Repository/UserRepository.php [hunk 3]
```diff
@@ -39,28 +41,31 @@ public function upgradePassword(PasswordAuthenticatedUserInterface $user, string
...
-//            ->orderBy('u.id', 'ASC')
-//            ->setMaxResults(10)
-//            ->getQuery()
-//            ->getResult()
-//        ;
-//    }
+    /**
+     * @return User[] Returns an array of User objects
+     */
+    public function findByExampleField(string $value): array
```
Comment (StoicFrenchDev) [inline src/Repository/UserRepository.php:47]: Are you using this method? If yes choose a different name

File: src/Repository/UserRepository.php [hunk 4]
```diff
@@ -39,28 +41,31 @@ public function upgradePassword(PasswordAuthenticatedUserInterface $user, string
...
-//            ->andWhere('u.exampleField = :val')
-//            ->setParameter('val', $value)
-//            ->getQuery()
-//            ->getOneOrNullResult()
-//        ;
-//    }
+    /**
+     * @throws NonUniqueResultException
+     */
+    public function findOneBySomeField(string $value): ?User
```
Comment (StoicFrenchDev) [inline src/Repository/UserRepository.php:62]: Same goes for this one. If not remove them

**Review by StoicFrenchDev — CHANGES_REQUESTED** [review]
Clean the dead code, and verify if you need the queries from the UserRepository.

**Comment by victorzanotelli** [comment]
> Clean the dead code, and verify if you need the queries from the UserRepository.

Thanks for your review, i finish what i'm doing rn and i'll fix it asap

**Review by StoicFrenchDev — APPROVED** [review]

**Pull request merged**